### ISSUE TITLE
Changelog

### ISSUE THREAD

**Issue by GRA0007** [confuser/graphql-constraint-directive#168]
Hi, forgive me if I've missed it somewhere but I can't seem to find a changelog for this project? We're currently using v3.1.1 and it looks like v5.0.0 is out now, but I'm reluctant to update because that's 2 breaking changes and no way to find out what if any changes we need to make to our code.

Is there a changelog somewhere I can reference to find out what the breaking changes were?

**Comment by confuser** [comment]
Yes https://github.com/confuser/graphql-constraint-directive/compare/v3.1.1...v5.0.0 but I'll keep this open to look at creating an automatic changelog file
